README.md

```diff
@@ -16,31 +16,54 @@ The **Platform.sh CLI** is the official command-line interface for [Platform.sh]
   * [Drush](https://github.com/drush-ops/drush) (for Drupal projects)
   * Other build tools: [npm](https://www.npmjs.com/), [pip](http://docs.python-guide.org/en/latest/starting/installation/), [bundler](http://bundler.io/), etc.
 
-## Installation
+## Install
 
-Run this command to install the CLI:
+To install the CLI, use either [Homebrew](https://brew.sh/) (on Linux, macOS, or the Windows Subsystem for Linux) or [Scoop](https://scoop.sh/) (on Windows):
 
-    curl -sS https://platform.sh/cli/installer | php
+### HomeBrew
 
-In some Windows terminals you may need `php.exe` instead of `php`.
+```console
+brew install platformsh/tap/platformsh-cli
+```
+
+### Scoop
+
+```console
+scoop bucket add platformsh https://github.com/platformsh/homebrew-tap.git
+scoop install platform
+```
+
+### Manual installation
+
+For manual installation, you can also [download the latest binaries](https://github.com/platformsh/cli/releases/latest).
 
-### Installing manually
+### Legacy installer
 
-1. Download the `platform.phar` file from the
-  [latest release](https://github.com/platformsh/platformsh-cli/releases/latest).
+_This installation method is considered legacy and is discouraged, use one of the methods above instead. Starting with version 5.x, this installation method will not be supported._
 
-2. Rename the file to `platform`, ensure it is executable, and move it into a
-  directory in your PATH.
+Run this command to install the CLI using the legacy installer, given that you have PHP already installed:
 
-3. Enable autocompletion and shell aliases:
+```console
+curl -sS https://platform.sh/cli/installer | php
+```
+
+In some Windows terminals you may need `php.exe` instead of `php`.
 
-        platform self:install
+## Upgrade
 
-## Updating
+Upgrade using the same tool:
 
-New releases of the CLI are made regularly. Update with this command:
+### HomeBrew
 
-    platform self:update
+```console
+brew upgrade platformsh-cli
+```
+
+### Scoop
+
+```console
+scoop update platform
+```
```

Comment:
~~~suggestion
```

### Legacy

```console
platform up
```
~~~

Reply:
I left this out, since the Legacy installer is mainly targeted to people that have the CLI already installed and know how to use it. At the end of the day, the Legacy installer supports auto-update.